users/views.py

```diff
@@ -39,3 +40,18 @@ class SearchUserView(generics.ListAPIView):
     filter_backends = (filters.SearchFilter,)
     queryset = User.objects.all()
     serializer_class = UserPublicSerializer
+
+
+class FollowerViewSet(viewsets.ModelViewSet):
+    permission_classes = (IsAuthenticated,)
+
+    def post(self, request, pk: int):
+        Followers.objects.update_or_create(follower=request.user, following_id=pk)
+        return Response(status=status.HTTP_201_CREATED)
+
+    def destroy(self, request, pk: int) -> Response:
+        to_delete = Followers.objects.filter(follower=request.user, following_id=pk)
```

Comment:
I did not mean that we have 2 different cases. I mean that we can return 200 or 204, both are ok. So you can turn the previous version back

Reply:
Done